cmd/podman/containers/clone.go

```diff
@@ -68,6 +71,10 @@ func clone(cmd *cobra.Command, args []string) error {
 			ctrClone.RawImageName = rawImageName
 		}
 	}
+	if ctrClone.Force && !ctrClone.Destroy {
+		return errors.Wrapf(define.ErrInvalidArg, "Cannot define --force without --destroy")
```

Comment:
errors should start lowecase

Reply:
I don't think we need to throw an error here.  What could be more helpful to users is to add `use --force to running container` in case `--destroy` is used without force.

Reply:
@vrothberg this is checking if `--force` was used without `--destroy` not the other way around. I guess I could set destroy to true implicitly is `--force` is used without it? I think it makes more sense here to error out if you are forcing a container to be removed that you did not actually set to be removed.